app/services/DatesService.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import connectors.DatesApiConnector
+import essttp.journey.model.{Journey, UpfrontPaymentAnswers}
+import essttp.rootmodel.CanPayUpfront
+import essttp.rootmodel.dates.InitialPayment
+import essttp.rootmodel.dates.extremedates.{ExtremeDatesRequest, ExtremeDatesResponse}
+import essttp.rootmodel.dates.startdates.{StartDatesRequest, StartDatesResponse}
+import play.api.mvc.RequestHeader
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.Future
+/**
+ * Essttp-dates Service.
+ */
+@Singleton
+class DatesService @Inject() (datesApiConnector: DatesApiConnector) {
+
+  def startDates(journey: Journey.AfterUpfrontPaymentAnswers)(implicit request: RequestHeader): Future[StartDatesResponse] = {
+    val startDatesRequest: StartDatesRequest = journey.upfrontPaymentAnswers match {
+      case j: UpfrontPaymentAnswers.DeclaredUpfrontPayment => ???
```

Comment:
is this to be filled in later or did you forget about this?

Reply:
It was as we aren't using it yet 😂